Twig/Extension/MediaExtension.php

```diff
@@ -169,7 +169,7 @@ public function thumbnail($media = null, $format, $options = array())
 
         $options = array_merge($defaultOptions, $options);
 
-        $options['src'] = $provider->generatePublicUrl($media, $format);
+        $options = $provider->getHelperProperties($media, $format, $options);
```

Comment:
That would be BC but would not fix #1065 : @navalex would still get the array to string conversion error, b/c the youtube providers adds an array `player_parameters` option, that is not compatible with the thumbnail template.